cmd/replace.go

```diff
@@ -17,40 +17,55 @@
 package cmd
 
 import (
-	"bufio"
-	"bytes"
 	"fmt"
 	"os"
-	"regexp"
 
 	"github.com/spf13/cobra"
 
-	"github.com/hacbs-contract/ec-cli/internal/image"
+	"github.com/hacbs-contract/ec-cli/internal/replacer"
 )
 
-type replaceFn func([]string, string) ([]byte, error)
+type replaceFn func([]string, string, *replacer.CatalogOptions) ([]byte, error)
 
 func replaceCmd(replace replaceFn) *cobra.Command {
 	var data = struct {
-		Source     string
-		Overwrite  bool
-		OutputFile string
-	}{}
+		Source           string
+		Overwrite        bool
+		OutputFile       string
+		CatalogName      string
+		CatalogRepoBase  string
+		CatalogHubAPIURL string
+	}{
+		CatalogName:      "tekton",
+		CatalogRepoBase:  "gcr.io/tekton-releases/catalog/upstream/",
+		CatalogHubAPIURL: "https://api.hub.tekton.dev",
+	}
 
 	cmd := &cobra.Command{
 		Use:   "replace",
 		Short: "Replace image references in the given input",
-		Example: `ec replace --source <source path> <image uri> [<image uri> ...]
+		Example: `ec replace --source <source path> [<image uri> ...]
+
+# display a modified version of the source file where
```

Comment:
We don't start with the comment for examples in other commands

Reply:
Ack. I wasn't aware. Updated.